scripts/sql/39_chart_ref.down.sql

```diff
@@ -0,0 +1,5 @@
+DELETE FROM "public"."chart_ref" WHERE ("location" = 'cronjob-chart_1-3-0' AND "version" = '1.3.0');
```

Comment:
39 migration is already merged in main, can you bump this please.

Reply:
the requested change has been made.